src/runtime/task/mod.rs

```diff
@@ -105,8 +108,14 @@ impl Task {
             name,
             span,
             local_storage: StorageMap::new(),
-            tag,
+            tag: tag.clone(),
```

Comment:
Seems like an unnecessary `clone` here if you're going to call `set_tag` below anyway